packages/ui/src/forum/components/threads/ThreadItem.tsx

```diff
@@ -156,3 +130,25 @@ export const ThreadItemWrapper = styled.a<{ $halfSize?: boolean }>`
     }
   }
 `
+
+const StyledThreadItem = styled.a`
```

Comment:
Also the goal of extracting `ThreadItemWrapper` is to have less code. So the common styles shouldn't be duplicated.

This looks looks good, so I think something this:
```ts
export const CardItem = styled.a`
  display: flex;
  flex-direction: column;
  gap: 16px;
  border: 1px solid ${Colors.Black[100]};
  border-radius: ${BorderRad.s};
  background-color: ${Colors.White};
  padding: 24px;
  &:hover,
  &:focus,
  &:focus-within {
    h5 {
      color: ${Colors.Blue[500]};
    }
  }
`
```
Should be extracted to a new file, e.g: `@/forum/components/CardItem`. And imported in CategoryListItem and ThreadItem, and if needed extended in there.